dist/index.html

```diff
@@ -9,44 +9,47 @@
   </head>
 
   <body>
-      <nav class="main-navbar">
-        <a href="" class="main-navbar__logo main-navbar__links">LOGO</a>
-        <a href="#home" class="main-navbar__links" id="home">home</a>
-        <a href="" class="main-navbar__links">about</a>
-        <a href="" class="main-navbar__links main-navbar__links-contact">contact</a>
-      </nav>
+    <nav class="main-navbar">
+      <a href="" class="main-navbar__logo main-navbar__links">LOGO</a>
+      <a href="#home" class="main-navbar__links" id="home">home</a>
+      <a href="" class="main-navbar__links">about</a>
+      <a href="" class="main-navbar__links main-navbar__links-contact"
+        >contact</a
+      >
+    </nav>
     <section class="banner">
       <img
         class="banner__image"
         src="images/norbert-levajsics-343107-unsplash.jpg"
-        alt="King kong figurine on desk beside computer"/>
+        alt="King kong figurine on desk beside computer"
+      />
       <div class="banner__text">
         <p class="banner__text-all banner__text-header">Lorem ipsum</p>
         <p class="banner__text-all">This is a demo site</p>
       </div>
     </section>
     <section class="portfolio">
-        <nav class="portfolio__navbar">
-          <a href="" class="portfolio__navbar-links">All</a>
-          <a href="" class="portfolio__navbar-links">lifestyle</a>
-          <a href="" class="portfolio__navbar-links">photo</a>
-          <a href="" class="portfolio__navbar-links">branding</a>
-        </nav>
+      <nav class="portfolio__navbar">
+        <a href="" class="portfolio__navbar-links">All</a>
+        <a href="" class="portfolio__navbar-links">lifestyle</a>
+        <a href="" class="portfolio__navbar-links ">photo</a>
+        <a href="" class="portfolio__navbar-links">branding</a>
+      </nav>
       <div class="portfolio__images">
-        <img 
+        <img
           class="portfolio__images-gallery"
           src="images/stephan-valentin-1415671-unsplash.jpg"
           alt="times square, New York"/>
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"/>
```

Comment:
Please name it correctly, like people names.

Reply:
Do you mean remove the numbers?

Reply:
No like this: "Times Square, New York".